### PR TITLE
RUMM-2590 Update distributed tracing on ios

### PR DESCRIPTION
**Pull request by maxep**
### What does this PR do?

Update the `Connect RUM and Traces` page for iOS:
- Specify the need for initialising the global `Tracer`
- Add a note for using `URLRequest`
- Fix code snippet indentation

### Motivation

Support Ticket

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by plousada — APPROVED** [review]

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 1]
```diff
@@ -135,14 +142,16 @@ To connect RUM to Traces, you need to specify your browser application in the `s
 
 **Note**: By default, all subdomains of listed hosts are traced. For instance, if you add `example.com`, you also enable tracing for `api.example.com` and `foo.example.com`.
```
Comment (alai97) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:143]:
```suggestion
By default, all subdomains of listed hosts are traced. For instance, if you add `example.com`, you also enable tracing for `api.example.com` and `foo.example.com`.
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 2]
```diff
@@ -135,14 +142,16 @@ To connect RUM to Traces, you need to specify your browser application in the `s
...
 
-4.  _(Optional)_ Set the `tracingSamplingRate` initialization parameter to keep a defined percentage of the backend traces. If not set, 100% of the traces coming from application requests are sent to Datadog. To keep 20% of backend traces:
+**Note 2**: Trace ID injection works when providing a `URLRequest` to the `URLSession`, distributed tracing will not work when using a `URL` object.
```
Comment (alai97) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:145]:
```suggestion
Trace ID injection works when you are providing a `URLRequest` to the `URLSession`. Distributed tracing does not work when you are using a `URL` object.
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 3]
```diff
@@ -135,14 +142,16 @@ To connect RUM to Traces, you need to specify your browser application in the `s
...
+
+5.  _(Optional)_ Set the `tracingSamplingRate` initialization parameter to keep a defined percentage of the backend traces. If not set, 100% of the traces coming from application requests are sent to Datadog. To keep 20% of backend traces:
```
Comment (alai97) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:147]:
```suggestion
5.  Optionally, set the `tracingSamplingRate` initialization parameter to keep a defined percentage of the backend traces. If not set, 100% of the traces coming from application requests are sent to Datadog. 

     To keep 20% of backend traces:
```

**Review by alai97 — CHANGES_REQUESTED** [review]

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 4]
```diff
@@ -116,15 +116,22 @@ To connect RUM to Traces, you need to specify your browser application in the `s
...
 
-3.  Initialize URLSession as stated in [Setup][1]:
+3. Initialize the global `Tracer`:
+    ```swift
+    Global.sharedTracer = Tracer.initialize(
+        configuration: Tracer.Configuration(...)
+    )
+    ```
+
+4.  Initialize URLSession as stated in [Setup][1]:
```
Comment (alai97) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:134]:
```suggestion
4. Initialize URLSession as stated in [Setup][1]:
```

**Review by alai97 — APPROVED** [review]
Thanks for opening the PR! I've added some formatting edits to make it consistent with the other Connect [Datadog] with Traces pages.

**Pull request merged**